<launch>

<arg name="controller_type" default="/moveit"/>
<arg name="camera_topic" default="/kinect2/qhd/image_color"/>
<arg name="camera_info_topic" default="/kinect2/qhd/camera_info"/>
<arg name="camera_optical_frame" default="/kinect2_rgb_optical_frame"/>
<arg name="leapmotion_frame" default="/teleop_left_frame"/>

<node name="leap_manager_node" pkg="yumi_manager" type="leap_manager_node" output="screen" respawn="false">
	<param name="controller_type" type="string" value="$(arg controller_type)"/>
	<param name="camera_topic" type="string" value="$(arg camera_topic)"/>
	<param name="camera_info_topic" type="string" value="$(arg camera_info_topic)"/>
        <param name="camera_topical_frame" type="string" value="$(arg camera_optical_frame)"/>
        <param name="leapmotion_frame" type="string" value="$(arg leapmotion_frame)"/>
	
</node>

</launch>
